frontend/components/events/ShareModal.tsx

```
@@ -108,25 +107,33 @@ ${eventURL}`;
+                    <div className="cursor-pointer" onClick={handleFacebookShare} title="Share on Facebook">
+                      <Image src={fb_icons} alt="Share via Facebook" className="h-11 w-11" />
+                    </div>
+                    <div className="cursor-pointer" onClick={handleEmailShare} title="Share via Email">
+                      <Image src={email_icons} alt="Share via Email" className="h-11 w-11" />
+                    </div>
                   </div>
+                  <p
+                    id="url"
+                    onClick={highlightURL}
+                    className="mt-4 cursor-pointer hover:bg-gray-50 p-2 rounded break-all"
+                  >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)